include/odp/api/spec/timer.h

```diff
@@ -114,6 +114,10 @@ void odp_timer_pool_param_init(odp_timer_pool_param_t *param);
  * The use of pool name is optional. Unique names are not required. Use odp_timer_pool_param_init()
  * to initialize timer pool parameters into their default values.
  *
+ * After creation a timer pool can be either started (see odp_timer_pool_start_multi()) or
+ * destroyed. The returned pool handle cannot be used with any other APIs before the pool is
+ * successfully started.
```

Comment:
Could mention odp_timer_pool_to_u64() as an exception, it could be used before start.